stdlib.go

```diff
@@ -135,16 +143,17 @@ const (
 )
 
 var (
-	logRegexp = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	StdlibLogRegexpDateTimeFileMsg = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
```

Comment:
```suggestion
	StdlibDefaultRegexp = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
```

Reply:
Although I wonder if StdlibRegexpDefault should correspond to the default stdlib log prefix, which is (AFAIK) just date+time? And then something like StdlibRegexpFull for all of these options.

Reply:
This would be a breaking change, i'm fine with that, should we go ahead here

Reply:
Good call. Let's not do it, then.